runtime_builders/publish_builders.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python
+
+# Copyright 2016 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import json
+import logging
+import subprocess
+import sys
+
+
+def main():
+    logging.getLogger().setLevel(logging.INFO)
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--config', '-c',
+                        help='builder config file',
+                        required=True)
+    args = parser.parse_args()
+
+    failures = 0
+    try:
+        with open(args.config, 'r') as f:
```

Comment:
Hmm, could this take a directory and walk, instead of a specific file? That will make it easier to run the job without having to pass in a parameter.